src/cloudsc_gpu/nlev_mod.F90

```diff
@@ -0,0 +1,3 @@
+MODULE NLEV_MOD
+  INTEGER, PARAMETER :: NLEV=137
```

Comment:
Should we have a sanity check somewhere in the inputs that validates this matches the hard-coded value?

Reply:
I was lazy doing it this way, it would not have been very difficult to pass NLEV down from driver call site.

Reply:
This one might actually go a bit deeper, as I actually tried to remove it and got CUDA runtime errors! Will need to re-create to give more details, but I don't think this is just about performance boost. I could be wrong, but I think the run-time was struggling with two-dimensional automatic arrays, only that increasing the the CUDA_HEAPSIZE did not help this time. I can have another look...

Reply:
Interesting, pretty sure I had it working without `nlev` as parameter, will look through my dev branch to see if I can find it.